initializr-service/src/main/resources/application.yml

```diff
@@ -477,6 +477,16 @@ initializr:
           groupId: io.zipkin.java
           artifactId: zipkin-server
           starter: false
+    - name: Cloud Task
+      bom: cloud-bom
+      versionRange: 1.3.6.RELEASE
+      content:
+        - name: Tasks
+          id: tasks
+          description: task result tracking along with integration with batch and streams
+          groupId: org.springframework.cloud
+          artifactId: spring-cloud-task-starter
+          version: 1.0.1.RELEASE
```

Comment:
If you have a jar that you don't mention in your own BOM id say that's a bug in Spring Cloud Task. We can't pick up a fix without a new release of Spring Cloud (SR4) though, and I'm not sure that's justified. Can you fix the bug, and we can make sure the starter is included in any future releases? Unfortunately that means we'all have to remember to come back and fix this as well.